CHANGELOG.md

```diff
@@ -9,3 +9,9 @@
 * [UICONSET-6](https://issues.folio.org/browse/UICONSET-6) Add tenant code to tenant profile.
 * [UICONSET-18](https://issues.folio.org/browse/UICONSET-18) "Switch active affiliation" profile dropdown action.
 * [UICONSET-125](https://issues.folio.org/browse/UICONSET-123) Add `ui-consortia-settings` app icon.
+* [UICONSET-7](https://issues.folio.org/browse/UICONSET-7) Create "Consortium manager" interface.
+* [UICONSET-8](https://issues.folio.org/browse/UICONSET-8) Limit access to "Consortium manager" interface.
```

Comment:
Could you please add UICONSET-8 to Related PRs?

Reply:
https://github.com/folio-org/ui-consortia-settings/pull/25
it's in description

Reply:
@Dmitriy-Litvinenko ^^ :)